bedrock/firefox/templates/firefox/features/picture-in-picture.html

```diff
@@ -6,110 +6,87 @@
+<h3>{{ ftl('features-pip-heres-how-it-works') }}</h3>
+<ol class="mzp-u-list-styled">
+  <li>{{ ftl('features-pip-play-any-video-in-your-updated') }}</li>
+  <li class="has-pip-icon">{{ ftl('features-pip-click-the-picture-in-picture-button') }}</li>
+  <li>{{ ftl('features-pip-cruise-around-to-other') }}</li>
+  <li>{{ ftl('features-pip-repeat-steps-1-3-to-have') }}</li>
+</ol>
 
-      <div class="mzp-l-columns mzp-t-columns-three mzp-t-picto-center">
-        {{ picto(
-          title=ftl('features-pip-watch-a-lecture-or-meeting'),
-          heading_level=4,
-          image=resp_img(
-            url='img/firefox/features/pip/pip-lecture.svg',
-            optional_attributes={
-              'class': 'mzp-c-picto-image',
-              'loading': 'lazy'
-            }
-          )
-        ) }}
-        {{ picto(
-          title=ftl('features-pip-keep-a-tutorial-video-open'),
-          heading_level=4,
-          image=resp_img(
-            url='img/firefox/features/pip/pip-cook.svg',
-            optional_attributes={
-              'class': 'mzp-c-picto-image',
-              'loading': 'lazy'
-            }
-          )
-        ) }}
-        {{ picto(
-          title=ftl('features-pip-entertain-cats-dogs-and-kids'),
-          heading_level=4,
-          image=resp_img(
-            url='img/firefox/features/pip/pip-entertain.svg',
-            optional_attributes={
-              'class': 'mzp-c-picto-image',
-              'loading': 'lazy'
-            }
-          )
-        ) }}
-      </div>
+<h3>{{ ftl('features-pip-3-more-ways-to-use-picture', fallback='features-pip-3-ways-to-use-picture') }}</h3>
 
-      <aside class="secondary-cta">
-      {{ download_firefox(dom_id='features-pip-bottom-download-secondary', download_location='secondary cta') }}
-      </aside>
-    </div>
-  </section>
+<div class="mzp-t-picto-center">
+  {{ picto(
+    title=ftl('features-pip-watch-a-lecture-or-meeting'),
+    heading_level=4,
+    image=resp_img(
+      url='img/firefox/features/pip/pip-lecture.svg',
+      optional_attributes={
+        'class': 'mzp-c-picto-image',
+        'loading': 'lazy'
+      }
+    )
+  ) }}
 
-{% endblock %}
+  {{ picto(
+    title=ftl('features-pip-keep-a-tutorial-video-open'),
+    heading_level=4,
+    image=resp_img(
+      url='img/firefox/features/pip/pip-cook.svg',
+      optional_attributes={
+        'class': 'mzp-c-picto-image',
+        'loading': 'lazy'
+      }
+    )
+  ) }}
+
+  {{ picto(
+    title=ftl('features-pip-entertain-cats-dogs-and-kids'),
+    heading_level=4,
+    image=resp_img(
+      url='img/firefox/features/pip/pip-entertain.svg',
+      optional_attributes={
+        'class': 'mzp-c-picto-image',
+        'loading': 'lazy'
+      }
+    )
+  ) }}
+</div>
+{% endblock article_content %}
 
 {% block js %}
   {{ js_bundle('firefox-features-pip') }}
```

Comment:
**Nit:** The JS bundle that prevents the Firefox CTA from showing up on Firefox browsers isn't working here because it's not inheriting `base-article.html`'s JS block. You just need to add `{{ super () }}` here

```suggestion
{% block js %}
  {{ super() }}
  {{ js_bundle('firefox-features-pip') }}
```